mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.kt

```diff
@@ -137,7 +137,7 @@ abstract class ContentController protected constructor(private val activity: Mai
         }
         noConnectionFragment?.let { page ->
             if (page.isAdded) {
-                fm.putFragment(state, "sitemapFragment", page)
+                fm.putFragment(state, "errorFragment", page)
```

Comment:
I'd use a constant here.

Reply:
Not sure. It's only the two methods next to each other (`onSaveInstanceState` and `onRestoreInstanceState`) that use this; introducing constants might be more cluttering than actual help. The bug in question happened via copy'n'paste, which also could have happened with constants...

Reply:
IMO constants should be used if the string is used more than once. This way it's pretty easy to find all usages of this string.